blocks.md

```diff
@@ -998,37 +997,36 @@ interface_name_include = []
 Key | Values | Required | Default
 ----|--------|----------|---------
 `primary_only` | Whether to show only the primary active connection or all active connections. | No | `false`
-`max_ssid_width` | Truncation length for SSID. | No | `21`
 `ap_format` | Acces point string formatter. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{ssid}"`
 `device_format` | Device string formatter. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{icon}{ap} {ips}"`
 `connection_format` | Connection string formatter. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{devices}"`
-`on_click` | On-click handler. Commands are executed in a shell. | No | `""`
 `interface_name_exclude` | A list of regex patterns for device interface names to ignore. | No | ""
 `interface_name_include` | A list of regex patterns for device interface names to include (only interfaces that match at least one are shown). | No | ""
 
 #### AP format string
 
-Placeholder | Description
-------------|-------------
-`{ssid}` | The SSID for this AP
-`{strength}` | The signal strength in percent for this AP
-`{freq}` | The frequency of this AP in MHz
+ Key | Value | Type
+-----|-------|-----
+`{ssid}` | The SSID for this AP | String
+`{strength}` | The signal strength in percent for this AP | Integer
+`{freq}` | The frequency of this AP in MHz | String
 
 #### Device format string
 
-Placeholder | Description
-------------|-------------
-`{icon}` | The icon matching the device type
-`{typename}` | The name of the device type
-`{name}` | The name of the device interface
-`{ap}` | The connected AP if available, formatted with the AP format string
-`{ips}` | The list of IPs for this device
+ Key | Value | Type
+-----|-------|-----
+`{icon}` | The icon matching the device type | String
+`{typename}` | The name of the device type | String
+`{name}` | The name of the device interface | String
+`{ap}` | The connected AP if available, formatted with the AP format string | String
+`{ips}` | The list of IPs for this device | String
 
 #### Connection format string
 
-Placeholder | Description
-------------|-------------
-`{devices}` | The list of devices, each formatted with the device format string
+ Key | Value | Type
+-----|-------|-----
+`{devices}` | The list of devices, each formatted with the device format string | String
+`{id}` | ??? | String
```

Comment:
What's this? An undocumented one you found?

Reply:
Yeah, it was undocumented and I decided to leave it as that.

Reply:
Let's just leave this one undocumented for now.